### ISSUE TITLE
Split help by locale

### ISSUE THREAD

**Issue by wjt** [flathub/org.gnome.Weather#18]
Currently all locales' helps are installed. This is 2.5 MB of the 8.3 MB app.

Ideally these would be either moved into the `.Locale` extension, which is already partially installed by locale, or to a separate `.Help` extension with locale subsetting.

**Comment by ghost** [comment]
I've never encountered this kind of usage. Do you have any examples of GNOME applications that use this approach?

**Comment by wjt** [comment]
Not offhand, but if we believe that splitting `.Locale` by locale makes sense, then surely splitting the (much larger!) help by locale does too?

4½ years ago, when I filed this, I was probably concerned with trying to reduce the size of Endless OS download images.

**Comment by ghost** [comment]
Actually, it would have been good for some projects, but we are probably too late to suggest it to Flatpak developers now.

**Comment by wjt** [comment]
I don't agree; but I would agree that Weather is not the right venue for that discussion, and I am no longer actively working on things like this.
